src/engine/strat_engine/backstore/devices.rs

```diff
@@ -112,7 +112,7 @@ pub fn get_devno_from_path(path: &Path) -> StratisResult<Device> {
 /// because they will not have the opportunity to be processed.
 pub fn find_stratis_devs_by_uuid(
     pool_uuid: PoolUuid,
-    uuids: Vec<DevUuid>,
+    uuids: &Vec<DevUuid>,
```

Comment:
Cippy has historically told me to write this as `&[DevUuid]` and not `&Vec<DevUuid>`. I'm not sure why it's not complaining now.

Reply:
Done.